index.md

```diff
@@ -25,49 +25,48 @@ Style guides and practices for writing code.
 
 ## Content
 
-Practices and recommendations for content.
+Practices and recommendations for content:
 
 - [Copy reviews](/practices/content/copy-reviews): A checklist for reviewing written content
 - [Filenames](/practices/content/filenames): Filename recommendations for files linked from websites
 
 ## Design
 
-Practices and recommendations for design.
+Practices and recommendations for design:
 
 - [Vanilla design specs](/practices/design/vanilla-design-specs): When to include when writing a design spec for a Vanilla pattern
 
-## [Foosball](/practices/foosball)
-
-Basic rules to achieve maximum enjoyment while playing of The Great Game.
-
-## [Podcasts](/practices/podcasts)
-
-Some suggested podcasts.
-
 ## Project-management
 
-Practices for managing projects within our squads.
+Practices for managing projects within our squads:
 
 - [Fibonacci estimation guide](/practices/project-management/fibonacci-estimation-guide): Guidelines for using points to estimate tasks
 - [Stand ups](/practices/project-management/stand-ups): How to run stand-ups in team squads
 - [Zenhub](/practices/project-management/zenhub): How to use Zenhub in team squads
 
+## Project structure
+
+Patterns & guidelines for structuring projects:
```

Comment:
Please use the word "and" instead of "&"

Ref: https://wiki.canonical.com/Marketing/Style_guide#Acronyms_and_abbreviations

Reply:
I didn't actually change this in this PR. But I'll fix it as a driveby I suppose.

Reply:
Fixed.